.github/workflows/codeql.yml

```diff
@@ -0,0 +1,48 @@
+name: CodeQL
+
+on:
+  pull_request:
+  workflow_dispatch:
+    branches: [ "master" ]
+
+jobs:
+  analyze:
+    if: github.repository == 'tarantool/tarantool'
```

Comment:
Let's extend this to 
 
```yml
    # Run inside tarantool/tarantool on workflow dispatch or pull request
    # if the 'notest' label is not set.
    if: github.repository == 'tarantool/tarantool' &&
        ( github.event_name != 'pull_request' ||
          ( github.event_name == 'pull_request' &&
            !contains(github.event.pull_request.labels.*.name, 'notest') ) )
```

for consistency with other workflows.

Reply:
Done.